cmd/tfimport/main.go

```diff
@@ -151,7 +151,18 @@ func planAndImport(rn, importRn runner.Runner) (retry bool, err error) {
 
 		// In dry-run mode, the output is the command to run.
 		if *dryRun {
-			importCmds = append(importCmds, output)
+			cmd := output
+			if err != nil {
+				cmd = err.Error()
+			} else {
+				// The last arg in import could be several space-separated strings. These need to be quoted together.
+				args := strings.Split(cmd, " ")
+				if len(args) >= 4 {
+					cmd = fmt.Sprintf("%v %v %v %q\n", args[0], args[1], args[2], strings.Join(args[3:], ""))
```

Comment:
You are joining the args[3:] with just empty string, is that intentional?

Reply:
Yes, but looking again I actually meant to use `SplitN`. I'll update that.